go/checks/checks_test.go

```diff
@@ -512,3 +512,31 @@ func TestCheckMaxPartitionsContributed(t *testing.T) {
 		}
 	}
 }
+
+func TestCheckConfidenceLevel(t *testing.T) {
+	for _, tc := range []struct {
+		desc            string
+		ConfidenceLevel float64
+		wantErr         bool
+	}{
+		{"negative confidenceLevel",
```

Comment:
1. formatting looks strange
2. {"confidenceLevel larger than 1",
			10,
			true}, can be in one line (it is too short)

Reply:
@MashaTelyatnikova  I followed the formatting from the previous tests, should i change it only for the ones i made? (switch to only one line)

Reply:
It shouldn't be always in different lines, it depends on the amount of args and length of the final lines (see example https://github.com/google/differential-privacy/blob/main/go/dpagg/sum_test.go#L199)

Reply:
so change it in the one you made